### DISCUSSION TITLE
[Esercitazione 6 Dicembre 2023] Esercizio 3

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#341] [Esercizi - Viaggi/Piazza] [answered]
Sia $A\in M_{34}(\mathbb{R})$ la matrice data da

$$\begin{pmatrix}2 & 1 & 3 & -1\\
1 & 1 & 1 & -2\\
-1 & 1 & -3 & -4\end{pmatrix}$$

e sia $L_A: \mathbb{R}^4\to \mathbb{R}^3$ l'applicazione lineare ad essa associata.
Scrivere l'espressione di $L_A:$

$$L_A \begin{pmatrix}x_1\\
x_2\\
x_3\\
x_4\end{pmatrix}= ....$$

Determinare una base per $Ker(L_A)$
Determinare la dimensione di $Im(L_A)$
Determinare una base per $Im(L_A)$
Studiare iniettività e suriettività di $L_A$

**Comment by Elia-Belli** [comment, marked as answer]
# Soluzione Ufficiale
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/403fc35a-5741-4285-85b6-0bb42af18f70)

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/86e0c048-6445-4bbb-8e5a-cbc7f637d52f)
